config/initializers/asset_sync.rb

```diff
@@ -0,0 +1,28 @@
+AssetSync.configure do |config|
+  config.fog_provider = 'AWS'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.